.goreleaser.yml

```diff
@@ -2,34 +2,84 @@ version: 2
 
 env:
   - GO111MODULE=on
+  - GOARM64=v8.0,lse,crypto
```

Comment:
Can you please add a comment why exactly lse, crypto

Reply:
LSE is not needed, but crypto is, as gemini is using crypto random generator, and if we have that ext on arm, we can use hardware acceleration

Reply:
Great, please add a comment